providers/msGraph/src/lib/msgraph.config.ts

```diff
@@ -0,0 +1,10 @@
+// eslint-disable-next-line @typescript-eslint/naming-convention
+export interface MSGraphConfig {
+  clientId: string;
+  authority: string;
+  secret: string;
+  scopes: string;
+  user: string;
+  fromAddress: string;
+  fromName: string;
```

Comment:
these three also could be defined as optional [reference](https://github.com/microsoftgraph/msgraph-sdk-javascript/blob/b720e7e2733cf9051628c21d21181eb2a8233bad/src/GraphRequest.ts#L647)
if `user` wont be defined then you will get API error